planner/languages/rust/rust_planner.go

```diff
@@ -33,9 +33,22 @@ func (p *Planner) IsRelevant(srcDir string) bool {
 }
 
 func (p *Planner) GetShellPlan(srcDir string) *plansdk.ShellPlan {
-	return &plansdk.ShellPlan{
+	plan := &plansdk.ShellPlan{
 		NixOverlays: []string{RustOxalicaOverlay},
 	}
+	manifest, err := p.cargoManifest(srcDir)
+	if err != nil {
+		return plan
+	}
+	rustVersion, err := p.rustOxalicaVersion(manifest)
+	if err != nil {
+		return plan
+	}
+
+	rustPkgDev := fmt.Sprintf("rust-bin.stable.%s.default", rustVersion)
+	plan.DevPackages = []string{rustPkgDev, "gcc"}
```

Comment:
A style nit is that its easier to track what fields are filled in the ShellPlan struct if we construct it at the end, as in:

```
manifest, err := ...
rustVersion, err := ...
rustPkgDev := fmt.Sprintf(....
return &plansdk.ShellPlan {
   DevPackages: []string{rustPkgDev, "gcc"}
   NixOverlays: ...
}
```

Reply:
Right, I only did copy pasting. Let me change the code a bit 👍